tests/test_foundations.py

```diff
@@ -17,21 +17,38 @@
 config = data.Configuration(
     atomic_numbers=molecule("H2COH").numbers,
     positions=molecule("H2COH").positions,
-    forces=molecule("H2COH").positions,
-    energy=-1.5,
-    charges=molecule("H2COH").numbers,
-    dipole=np.array([-1.5, 1.5, 2.0]),
+    properties={
+        "forces": molecule("H2COH").positions,
+        "energy": -1.5,
+        "charges": molecule("H2COH").numbers,
+        "dipole": np.array([-1.5, 1.5, 2.0]),
```

Comment:
Pretty much this could be copied over to test_data.py

Reply:
true, but this is not anything I have touched